scout/adapter/mongo/base.py

```diff
@@ -120,5 +121,17 @@ def collections(self):
         """
         return self.db.list_collection_names()
 
+    def collection_stats(self, coll_name):
+        """Returns stats from a single collection
+
+        Args:
+            coll_name: name of collection to retrieve stats for
+
+        Returns:
+            stats(dict): dictionary with collection stats
+        """
+        db = current_app.config.get("MONGO_DATABASE")
+        return db.command("collstats", coll_name)
```

Comment:
Does this really return a dictionary or is it a pymongo object?

Reply:
Dictionary. Using a  type(db.command("collstats", coll_name)) debug message:

![image](https://user-images.githubusercontent.com/28093618/100454254-d2969180-30bc-11eb-8802-56f9fdfcb7d1.png)